public/app/features/browse-dashboards/components/BrowseActions.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useMemo } from 'react';
+
+import { Input } from '@grafana/ui';
+import { ActionRow } from 'app/features/search/page/components/ActionRow';
+import { SearchLayout } from 'app/features/search/types';
+
+export default function BrowseActions() {
```

Comment:
minor nit: think we [prefer named exports](https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#exports)